lockfile_test.go

```diff
@@ -1,28 +1,112 @@
-package lockfile_test
+package lockfile
 
 import (
-	lockfile "."
 	"fmt"
-	"os"
+	"io/ioutil"
+	"math/rand"
 	"path/filepath"
+	"strconv"
+	"testing"
 )
 
-func ExampleLockfile() {
```

Comment:
I'm afraid I don't really know what you're talking about. I have tried following the documentation on writing test functions for the library (https://golang.org/pkg/testing/) as much as I can.

I'll put the old test file back, but I'm struggling so see the reasoning behind deviating form the go standard. Am I missing something?